CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,93 @@
+expression, marital status, military or veteran status, nationality, personal
+appearance, race, religion, sexual orientation, socioeconomic status, tribe, or
+any other dimension of diversity.
+
+## Scope
+
+This code of conduct applies:
+
+- within project and community spaces,
+- in other spaces when an individual community participant's words or actions
+  are directed at or are about a project, our community, or another community
+  participant.
+
+## Our Standards
+
+Our Community is open, inclusive and respectful. Every member of our community
+has the right to have their identity respected.
+
+Examples of behavior that contributes to a positive environment include but are
+not limited to:
+
+- Demonstrating empathy and kindness toward other people
+- Being respectful of differing opinions, viewpoints, and experiences
+- Giving and gracefully accepting constructive feedback
+- Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+- Focusing on what is best not just for us as individuals, but for the overall
+  community
+- Using welcoming and inclusive language
+
+Examples of unacceptable behavior include but are not limited to:
+
+- The use of sexualized language or imagery
+- Trolling, insulting or derogatory comments, and personal or political attacks
+- Public or private harassment in any form
+- Publishing others' private information, such as a physical or email address,
+  without their explicit permission
+- Violence, threatening violence, or encouraging others to engage in violent
+  behavior
+- Stalking or following someone without their consent
+- Unwelcome physical contact
+- Unwelcome sexual or romantic attention or advances
+- Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+The following behaviors are also prohibited:
+
+- Providing knowingly false or misleading information in connection with a Code
+  of Conduct investigation or otherwise intentionally tampering with an
+  investigation.
+- Retaliating against a person because they reported an incident or provided
+  information about an incident as a witness.
+
+Project maintainers have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct. By adopting this Code of Conduct,
+project maintainers commit themselves to fairly and consistently applying these
+principles to every aspect of managing a project. Project maintainers who do
+not follow or enforce the Code of Conduct may be temporarily or permanently
+removed from the project team.
+
+## Reporting
+
+For incidents occurring in our community, please contact any of the individual
+maintainers of the GitHub organization or Slack workspace.
+
+## Enforcement
+
+Upon review and investigation of a reported incident, the CoC response team
```

Comment:
nit: we don't have a CoC respone team to do we? maybe just make this "AEP maintainers".